setup.py

```diff
@@ -18,7 +18,7 @@
 
 setup(
     name="mbed-cli",
-    version="1.8.3",
+    version="1.8.4",
```

Comment:
It this supposed to be here?

Reply:
Yes. Read the description. This was supposed to get in first for a quick 1.8.4 release to hotfix the issue. Due to the new feature now merged in, this will have to become 1.9.0